### PR TITLE
fix: log taskkill error based on logging opts (#178)

### LINKED ISSUE #178: "Process could not be terminated" error is thrown on Windows
First of all, thanks for this awesome library! I use it together with puppeteer-core to avoid chromium installs if Chrome exists on users system, in the library that I maintain: https://github.com/onderceylan/pwa-asset-generator

I'm getting the following error on Windows, after calling `chrome.kill()`. 

```
ERROR: The process with PID 12345 (child process of PID 1234) could not be terminated.
Reason: There is no running instance of the task.
```

I see 2 issues with this;

1. Not sure why but sometimes this error occur and sometimes not. Ideally, I hope the root cause of this can be found and fixed.

2. Second issue is, this error is being logged to the console directly, and logLevel setting is not respected when this error is thrown. Ideally, it should be logged based on logLevel setting.

I hope it's possible to get this issue fixed. If not, I'd love to go for the second option as this issue logs an unnecessary output for my CLI lib at the moment.

### PR DESCRIPTION
**Pull request by onderceylan**
fixes #178

### CODE REVIEW

**Comment by onderceylan** [comment]
@googlebot I signed it!

File: src/chrome-launcher.ts [hunk 1]
```diff
@@ -332,7 +332,7 @@ class Launcher {
         log.log('ChromeLauncher', `Killing Chrome instance ${this.chrome.pid}`);
         try {
           if (isWindows) {
-            execSync(`taskkill /pid ${this.chrome.pid} /T /F`);
+            execSync(`taskkill /pid ${this.chrome.pid} /T /F`, {stdio: 'pipe'});
```
Comment (patrickhulce) [inline src/chrome-launcher.ts]: add a comment that basically says while pipe is the default, stderr also gets printed to process.stderr if you don't explicitly set it? I could see someone forgetting this and removing it in a driveby someday :)

Comment (onderceylan) [reply src/chrome-launcher.ts]: Sure, feels safer :)

**Comment by onderceylan** [comment]
Hi @patrickhulce, I made the requested changes and rebased the commits. It's ready for re-review.

File: src/chrome-launcher.ts [hunk 2]
```diff
@@ -332,7 +332,9 @@ class Launcher {
         log.log('ChromeLauncher', `Killing Chrome instance ${this.chrome.pid}`);
         try {
           if (isWindows) {
-            execSync(`taskkill /pid ${this.chrome.pid} /T /F`);
+            // While pipe is the default, stderr also gets printed to process.stderr
+            // if you don't explicitly set it
```
Comment (patrickhulce) [inline src/chrome-launcher.ts]:
```suggestion
            // if you don't explicitly set `stdio`
```

Comment (onderceylan) [reply src/chrome-launcher.ts]: Done

**Pull request merged**